README.md

```diff
@@ -27,7 +27,22 @@ Also, Noise calls in low depth region are removed using EM algorithm.
  limitations under the License.
 
 
+# Build
 
+## Prerequisites
+
+- git

```

Comment:
```suggestion
```

Reply:
We need git not only for cloning the repository but for executing `./gradlew uberjar`

Reply:
Sorry. I don't know Gradle uses Git internally :open_mouth: